test/e2e/volume_provisioning.go

```diff
@@ -125,6 +125,38 @@ var _ = framework.KubeDescribe("Dynamic provisioning", func() {
 
 			testDynamicProvisioning(c, claim)
 		})
+
+		It("should test that deleting a claim before the volume is provisioned deletes the volume. [Slow][Volume]", func() {
+			// This case tests for the regressions of a bug fixed by PR #21268
+			// REGRESSION: Deleting the PVC before the PV is provisioned can result in PV
```

Comment:
This was a race condition, i.e. perfect timing of PVC deletion in the middle of provisioning was required. By creating just one PVC, the probability of catching this is really low. What about creating 10 - 20 (- 100?) PVCs?